src/6_branch.js

```diff
@@ -459,9 +459,10 @@ Branch.prototype['init'] = wrap(
 		};
 
 		if (sessionData && sessionData['session_id'] && !link_identifier) {
+			// resets data in session storage to prevent previous link click data from being returned to Branch.init()
+			session.update(self._storage, { "data": "" });
```

Comment:
@yntema thanks for noticing this. The reason why we want to make it an empty string is because most partners use the cdn version of the WebSDK which is always up-to-date with the latest code. In the past we have returned the previous session data which is a non-empty string. If we return a null, this could cause some sites throw errors.